shared/src/types/address.rs

```diff
@@ -298,6 +299,20 @@ pub fn kartoffel() -> Address {
     Address::decode("a1qq5qqqqqxs6yvsekxuuyy3pjxsmrgd2rxuungdzpgsmyydjrxsenjdp5xaqn233sgccnjs3eak5wwh").expect("The token address decoding shouldn't fail")
 }
 
+/// Temporary helper for testing, a hash map of tokens addresses with their
+/// informal currency codes.
+pub fn tokens() -> HashMap<Address, String> {
```

Comment:
this could also return a `HashMap<Address, &'static str>`  which helps with borrow semantics elsewhere in the code base. You can also acheive this code in a single allocation with 
```rust
 vec![(xan(), "XAN"),
        (btc(), "BTC"),
        (eth(), "ETH"),
        (dot(), "DOT"),
        (schnitzel(), "Schnitzel"),
        (apfel(), "Apfel"),
        (kartoffel(), "Kartoffel")]
     .into_iter()
     .collect()
 ```

Reply:
that's better, thanks! Done in cd8b6c1